bindata/network/ovn-kubernetes/common/007-rbac-cluster-reader.yaml

```diff
@@ -11,6 +11,7 @@ rules:
   - egressfirewalls
   - egressips
   - egressqoses
+  - egressservices
```

Comment:
Any reason to leave out `egressservices/status` here? I thought it wouldn't be specifically needed for read verbs but I see it added elsewhere just for read verbs anyway. I might be missing something.

Reply:
It is the other way around, there is no need for the reader roles to have `egressservices/status` as well as `egressservices`.
Pushed a fix.